R/bundle.R

```diff
@@ -937,13 +957,15 @@ performPackratSnapshot <- function(bundleDir) {
   }
 
   # generate a snapshot
-  suppressMessages(
+  # suppressMessages(
```

Comment:
Why is this commented out? Do we want to see these messages even if `verbose` is turned off?

Reply:
Fixed in latest commit